lib/data/external/GCP/GcpService.js

```diff
@@ -1,11 +1,40 @@
+const googleAuth = require('google-auto-auth');
 const AWS = require('aws-sdk');
 const { errors } = require('arsenal');
 const Service = AWS.Service;
 
 const GcpSigner = require('./GcpSigner');
 
+function genAuth(authParams) {
+    const authOptions = authParams || {};
+    Object.assign(authOptions, {
+        scopes: 'https://www.googleapis.com/auth/cloud-platform',
```

Comment:
Please move this to config. It may not need modifying but it's worth making it flexible to configure.

Reply:
Understood. I will move the hard coded scopes value as a locationConstraint value.